engine/lib/phx/src/math/intersect.rs

```diff
@@ -83,14 +83,14 @@ pub unsafe extern "C" fn Intersect_PointTriangle_Barycentric(p: &Vec3, tri: &Tri
 }
 
 #[no_mangle]
-pub extern "C" fn Intersect_RayPlane(ray: &Ray, plane: &Plane, pHit: &mut Vec3) -> bool {
+pub extern "C" fn Intersect_RayPlane(ray: &Ray, plane: &Plane, pHit: &mut Position) -> bool {
```

Comment:
I see in this function we do convert coordinates to and from Position. Can we do everything in Position coordinates instead for consistency and higher precision?

Reply:
I did this as a little bit of a hack for now, to avoid having to rewrite much of the BSP code which uses Plane's and Ray's extensively. I'm not sure that the BSP code is even used anymore, so we might end up removing it before changing it.